pkg/console/auth.go

```diff
@@ -70,8 +70,7 @@ func (c *Client) authenticator(cl *stdsdk.Client, res *http.Response) (http.Head
 	}
 
 	if err := cl.Post(m[1], ro, &s); err != nil {
-		fmt.Printf("err: %+v\n", err)
```

Comment:
should we still print the error?

Reply:
The error will be returned to the user. The old behavior was to print the error but it was not failing.